knowlg-app/src/app/app.module.ts

```diff
@@ -8,11 +8,17 @@ import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app-routing.module';
 import { SunbirdPdfPlayerModule } from '@project-sunbird/sunbird-pdf-player-v9';
 import { SunbirdEpubPlayerModule } from '@project-sunbird/sunbird-epub-player-v9';
-
+import { QuestionCursor } from '@project-sunbird/sunbird-quml-player-v9';
```

Comment:
Question cursor is optional cross it

Reply:
As i am  getting No provider for QuestionCursor , because if it i have added.